src/binary/ibc-setup/commands/ics20.ts

```diff
@@ -136,15 +136,27 @@ async function resolveLink(
       connections.dest,
       logger
     );
-    logger.info(
-      `Used existing connections ${link.endA.connectionID} (${link.endA.clientID}) <=> ${link.endB.connectionID} (${link.endB.clientID})`
+    console.log(
+      `Used existing connections [chain_id: ${link.endA.chainId()}, connection_id: ${
```

Comment:
Good to be explicit, but since connectionID is always `connection-XX` and client_id always `07-tendermint-XX`, not sure if we need all these labels. You can remove them, or merge as is and decide to trim down if it is overwhelming.